### PR TITLE
Show statistic.

### PR DESCRIPTION
**Pull request by drets**
• introduce flow
• hide icons on search or adding new page in the navbar
• show statistic popup on the main page
• show statistic on the login page

### CODE REVIEW

File: src/containers/HeaderBar/HeaderBar.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ import { rememberArticles } from './../../actions/articles';
 import { isJsonString } from './../../utils/jsonUtils';
 import { getViewState } from './../../selectors/view';
 import { getArticles } from './../../selectors/articles';
-
+import Star from './_components/Star/index';
```
Comment (rasendubi) [inline src/containers/HeaderBar/HeaderBar.js:30]: The module name is strange. Why `_components` with an underscore?

Comment (rasendubi) [reply src/containers/HeaderBar/HeaderBar.js:30]: What is Star?

Comment (rasendubi) [reply src/containers/HeaderBar/HeaderBar.js:30]: Ok, I understand the stats component uses a star icon but the component name is completely off.

Comment (drets) [reply src/containers/HeaderBar/HeaderBar.js:30]: :+1:

Comment (drets) [reply src/containers/HeaderBar/HeaderBar.js:30]: https://github.com/heutagogy/heutagogy-frontend/pull/101/commits/21970490a65c986ac09d2ced553719d29c05eb28

**Pull request merged**